pkg/controller/controller.go

```diff
@@ -149,6 +150,7 @@ func (c *Controller) Run() error {
 				// TODO: set version in spec in apiserver
 				s.Version = defaultVersion
 			}
+			clus.Spec.Version = strings.Trim(clus.Spec.Version, "v")
```

Comment:
well. ok. if you want it works this way, can you create a func in spec pkg: spec.clean() or something like that? also move the version defaulting into it.